private/buf/cmd/buf/command/push/push.go

```diff
@@ -165,6 +167,12 @@ If you set the --%s flag and/or --%s flag yourself, then the value(s) will be us
 			createDefaultLabelFlagName,
 		),
 	)
+	flagSet.BoolVar(
+		&f.SkipUnnamed,
+		skipUnnamedFlagName,
+		false,
+		"Skip unnamed modules will only push named modules. Named modules must not have any unnamed dependencies.",
```

Comment:
This doc comment is kind of confusing to parse. What about this?
```suggestion
		"Only push named modules to the BSR. Named modules must not have any unnamed dependencies.",
```